epcpy/utils/regex.py

```diff
@@ -163,3 +163,69 @@
     f"|{SGCN_GS1_ELEMENT_STRING}|{GINC_GS1_ELEMENT_STRING}|{GSIN_GS1_ELEMENT_STRING}"
     f"|{ITIP_GS1_ELEMENT_STRING}|{UPUI_GS1_ELEMENT_STRING}|{PGLN_GS1_ELEMENT_STRING})"
 )
+
+# Binary headers
+BINARY_HEADER_GDTI_96 = "00101100"
+BINARY_HEADER_GSRN_96 = "00101101"
+BINARY_HEADER_GSRNP = "00101110"
+BINARY_HEADER_USDOD_96 = "00101111"
+BINARY_HEADER_SGTIN_96 = "00110000"
+BINARY_HEADER_SSCC_96 = "00110001"
+BINARY_HEADER_SGLN_96 = "00110010"
+BINARY_HEADER_GRAI_96 = "00110011"
+BINARY_HEADER_GIAI_96 = "00110100"
+BINARY_HEADER_GID_96 = "00110101"
+BINARY_HEADER_SGTIN_198 = "00110110"
+BINARY_HEADER_GRAI_170 = "00110111"
+BINARY_HEADER_GIAI_202 = "00111000"
+BINARY_HEADER_SGLN_195 = "00111001"
+BINARY_HEADER_ADI_VAR = "00111011"
+BINARY_HEADER_CPI_96 = "00111100"
+BINARY_HEADER_CPI_VAR = "00111101"
+BINARY_HEADER_GDTI_174 = "00111110"
+BINARY_HEADER_SGCN_96 = "00111111"
+BINARY_HEADER_ITIP_110 = "01000000"
+BINARY_HEADER_ITIP_212 = "01000001"
```

Comment:
These headers are already defined in the TagEncodable classes. We shouldn't define such things twice, maybe do something like https://github.com/nedap/retail-epcpy/blob/a5f7ee14e97170448ba0c89cd24b1a9990cd4c2c/epcpy/utils/parsers.py#L103

Reply:
Agreed